\documentclass[12pt]{amsart}
\usepackage[margin=1in]{geometry}                % See geometry.pdf to learn the layout options. There are lots.
\usepackage{color}
\geometry{letterpaper}                   % ... or a4paper or a5paper or ... 
%\geometry{landscape}                % Activate for for rotated page geometry
%\usepackage[parfill]{parskip}    % Activate to begin paragraphs with an empty line rather than an indent



%\usepackage[margin=0.5in]{geometry}

\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{epstopdf}
\newcommand{\mbh}[1]{\textcolor{red}{ \emph{\scriptsize  #1}} }
\DeclareGraphicsRule{.tif}{png}{.png}{`convert #1 `dirname #1`/`basename #1 .tif`.png}

\title{NSF-GRFP: Research Statement}
\author{David E. Hufnagel}
%\date{}                                           % Activate to display a given date or no date

\begin{document}
\maketitle
%\section{}
%\subsection{}

\textbf{\underline{Introduction:} }
Hybridization is an underappreciated evolutionary force.
Studies suggest most angiosperms have polyploidy somewhere in their history (Masterson 1994) which means that many of these organisms have hybrid ancestors. \mbh{but polyploidy is not a prerequisite for hybridization}
Hybridization improves adaptability through new combinations of alleles and increased heterozygosity \textbf{(cite plant that took over england)}. \textbf{(hybrid zones are interesting)}
%Hybridization can have a substantial influence on whether native species can survive invasion of related species. 
%n some cases genetic mixing can remove most or all pure members of a species like with red wolves \textbf{(cite)} but sometimes a native species survival of an invasion is dependent on introgression from the invader like in the case of californian salamanders \textbf{(cite)}.

%The main focus of this project is the affects of admixture on populations of Zea, the genus containing maize.  \mbh{I think you need a more general introduction here.  What doe the literature say about the significance of hybridization and introgression in the evolution of species?  How are domesticated species and their wild relatives unique opportunities to study hybridization and introgression given their history and relationship to each other?}
%Maize is the ideal system for this study not only because it is one of the most important world food crops \textbf{(citation)}, but also because it is spread over a wide area with many locally adapted populations.  \mbh{you'll need to be careful to distinguish between maize and teosinte...are we looking at local adaptation and gene flow in maize or in teosinte?}
Teosinte is an ideal system for studying hybridization and hybrid zones.  
Although teosinte represents all wild relatives of maize in the genus {Zea}, for the purposes of this document \textit{Zea mays ssp. parviglumis} (hereafter parviglumis) and \textit{Zea mays ssp. mexicana} (hereafter mexicana) in aggregate will be called teosinte. 
Maize is the world's most important food crop as well as a well studied system with many available genetic resources.
These resources can be used to aid research in teosinte, which can in turn be used to improve our knowledge of maize.
One reason teosinte is a good study system is that the habitats of parviglumis and mexicana are easily distinguished by their altitude with hybrids inhabiting intermediate altitudes between the two subspecies.
The genome size and complexity of teosinte is roughly average for angiosperms, making it a good representative of this clade (Gregory 2007). 
Parviglumis and mexicana diverged recently, yet there are clear signs of local adaptation as well as ongoing hybridization between each other and with maize within hybrid zones (Fukunaga 2005).
Previous studies suggest some of teosinte's local adaptations have been conferred to maize during it's spread across the Americas following domestication \textbf{(cite)}.

I have identified three zones of clustered Mexican hybrid populations between lowland parviglumis and highland mexicana in the Central Plateau of Mexico (CPM) and in the northern and southern Balsas River Valley of Mexico (BRVM). 
To identify hybrids, I analyzed an available Single Nucleotide Polymorphism (SNP) dataset  (Heerwaarden 2010, Heerwaarden 2011, Fang 2012) including 983 SNPs genotyped in 2,793 individuals from all species and subspecies of \textit{Zea} across the Americas as well as members of the genus \textit{Tripsacum}.
To identify hybrids, I have analyzed the SNP dataset using the program STRUCTURE.
This program uses a model-based approach to infer population structure and assign individuals to populations using multilocus genotype data (Pritchard 2000).
Samples of majority attribution to one teosinte with a significant ($\geq$ 25\%) attribution to the other are considered to be hybrids.
I plan to confirm these identifications using Reich's f$_{\text{3}}$ ancestry estimation method (Reich  2011).  
This method provides SNP-by-SNP estimates of admixture and has an advantage over STRUCTURE in that it is robust to ascertainment bias, which is likely an issue with our dataset as the SNPs were designed exclusively in maize, but SNP-Chip was used for a broad range of Zea individuals.

Hybrid zones often form on the borders between parapatric populations of closely related populations regardless of whether there is a selective advantage to the hybrid phenotype.
Teosinte hybrid zones have never been deeply studied before, although one hybrid population was mentioned and identified as admixed (Pyhajarvi 2013)  and others have been briefly mentioned (Fukunaga 2005).
My expectation is that the width of these hybrid zones is largely determined by the steepness of the altitudinal gradient because the niches of parviglumis and mexicana are heavily dependent on altitude.
 \textbf{(more hybrid zone talk)} 


To better understand these hybrids I have formed 3 objectives: 
\begin{enumerate} 
	\item Determine how these hybrids are distributed across Mexico, the widths of the hybrid zones they reside in and how those widths compare to the expected widths.
	\item Ascertain the relationship of these hybrids with each other and their neighboring teosinte.
	\item Explore whether these zones contain stable, locally adapted populations or are simply a product of ongoing hybridization between neighboring teosinte as in a tension zone.
\end{enumerate} 

\hfill\break \textbf{\underline{Objective 1: How extensive are teosinte hybrid zones?} } 
In addition to the SNP dataset that is already available, I would like to generate Genotyping By Sequencing data (GBS) of hybrid individuals as well as members of nearby populations of teosinte to answer more in-depth questions about the history of these hybrid zones as well as the nature and degree of admixture amongst the hybrids and between the hybrid populations and their neighbors.  
While we could use the same SNP-chip used to obtain our SNP dataset on new samples, GBS provides much higher resolution data at a lower cost.
In order to acquire the GBS data I will likely  need to do some sampling in the regions where these hybrid populations reside.  
Due to the high crime rates in both BRVM regions and the recent Guerrero student massacre I will only collect samples in the CPM, leaving sampling in more dangerous areas to our colleagues working in Mexico who have more local knowledge and experience working in these regions.

One use for the GBS data would be sampling in a transect of the hybrid zones to investigate the proportion of hybrids at various altitudes.  
Additionally, our collaborators in Mexico are currently gathering seed in all three zones so that we will have more individuals to analyze for this study.
Another topic of interest is the observed and expected hybrid zone width. \textbf{(more hybrid zone talk)}

\hfill\break \textbf{\underline{Objective 2: Is the genomic architecture of hybridization conserved?} } To determine the relationship of these hybrids with each other and their ancestors, I plan to determine the diversity of the zones using measures of heterozygosity as well as the differentiation of the zones using Wright's Fst.  
Together these will give us a rough idea of within and between zone relatedness.
I also plan to use the D statistic from (Durand 2011) to determine whether these hybrids are sister to either teosinte, ancestral to both or are true hybrids showing equal clustering with each teosinte.
It would also be interesting to use the GBS data to see whether the genetic architecture vary amongst populations within and between zones.
Understanding the genetic architecture could reveal whether there are conserved regions amongst different zones or at particular altitudes suggesting that they have been selected for within that niche.

\hfill\break \textbf{\underline{Objective 3: Is the hybrid phenotype adaptive?} }To answer my question about the origin and stability of the hybrid zones, one measure I plan to use is the relative fitness of the hybrids based on a common garden experiment in the CPM. 
If these zones contain stable, locally adapted populations that are fitting into a niche they should not only be present in the intermediate altitude, but also be more fit there and be less fit in higher and lower altitudes relative to mexicana and parviglumis respectively.
I would also like to use HapMix (Price 2009) with our GBS data to build a histogram of the lengths of ancestry segments.
As ancestry segments of a hybrid individual should break up over time due to recombination, if these populations resulted from one or more hybridization events close in time there should be a peak of ancestry segments near a specific length.
If these hybrids are the result of ongoing hybridization the histogram should look roughly like an exponential decay graph as most ancestry segments would be highly broken up and some would be longer.% indicating many hybridization events including recent events.

To build a tree of these hybrids and proximal teosinte populations I plan to use a software called Treemix (Pickrell 2012).
Phylogenetic trees, while useful, are often an oversimplification of the relationships between populations. % which can include migrations between populations, hybridization between individuals and other events which cannot be represented on a traditional phylogenetic tree.
Treemix improves on these trees by adding directed and weighted migration edges. % to improve the statistical likelihood of the tree.
Along with a STRUCTURE analysis of these groups and their neighboring teosinte, these analyses should paint a clear picture of the introgression history amongst these hybrid groups including whether their ancestry was originally more parviglumis or mexicana and whether some hybrid populations are derived from others.

\hfill\break \textbf{\underline{Resources Needed:} } To successfully complete this project I will require funding for my salary, for GBS and for travel to Mexico.  I will also require the assistance of my major professor Matthew Hufford as well as our collaborators at UC Davis and in Mexico.
A benefit of my work being largely computational and using an available dataset is low data generation costs.

\hfill\break \textbf{\underline{Broader Impacts:} }
To reach out to the greater community I plan to participate in Iowa State University's GK12 program.  Through the program I will be sharing my research with children from the Des Moines public school system.  
I expect that my work will interest these students as Iowa's agriculture is dominated by corn.
The Des Moines public school system is the most diverse in the state of Iowa in terms of representing ethnic minorities.  This program is therefore a great opportunity to get schoolchildren, including underprivileged groups, excited about STEM research.  
%I believe that I am uniquely capable of answering these questions because I have experience in programming and computational analyses, experienced advisors and collaborators, a solid academic foundation in genomics and the drive and curiosity to stay focused on the project.

%Bibliography
\hfill\break \textbf{\underline{Works cited:}}
\small{ 
\textbf{1: Durand} et al. (2012). \textit{Molecular Biology and Evolution}, doi: 10.1093/molbev/msr048.
\textbf{2: Fang} et al. (2012). \textit{Genetics}, doi: 10.1534/genetics.112.138578. 
\textbf{3: Fukunaga} et al. (2005). \textit{Genetics}, doi: 10.1534/genetics.104.031393. 
\textbf{4: Gregory} et al. (2007). \textit{Nucleic Acids Research}, doi:10.1093/nar/gkl828. 
\textbf{5: Heerwaarden} et al. (2010). \textit{Molecular Ecology}, 19(6) 1162-1173. 
\textbf{6: Heerwaarden} et al. (2011). \textit{PNAS}, 108(3) 1088-1092.  
\textbf{7: Masterson} (1994). \textit{Science}, 264(5157) 421-424.  
\textbf{8: Pickrell} (2012). \textit{PLOS Genetics}, DOI: 10.1371/journal.pgen.1002967. 
\textbf{9: Price} (2009). \textit{PLOS Genetics}, DOI: 10.1371/journal.pgen.1000519. 
\textbf{10: Pritchard} (2000). \textit{the Genetics Society of America}, 155: 945-959.  
\textbf{11: Pyhajarvi} et al. (2013). \textit{Genome Biology and Evolution}, 264(5157) 421-424.  
\textbf{12: Reich} et al. (2011). \textit{The American Journal of Human Genetics}, 89 516-528.

}

\end{document}  