\documentclass[12pt]{amsart}
\usepackage[margin=1in]{geometry}                % See geometry.pdf to learn the layout options. There are lots.
\usepackage{color}
\geometry{letterpaper}                   % ... or a4paper or a5paper or ... 
%\geometry{landscape}                % Activate for for rotated page geometry
%\usepackage[parfill]{parskip}    % Activate to begin paragraphs with an empty line rather than an indent



%\usepackage[margin=0.5in]{geometry}

\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{epstopdf}
\newcommand{\mbh}[1]{\textcolor{red}{ \emph{\scriptsize  #1}} }
\DeclareGraphicsRule{.tif}{png}{.png}{`convert #1 `dirname #1`/`basename #1 .tif`.png}

\title{Graduate Research Plan Statement: David E. Hufnagel}
%\author{David E. Hufnagel}
%\date{}                                           % Activate to display a given date or no date

\begin{document}
\maketitle
%\section{}
%\subsection{}

\textbf{\underline{Introduction:} }
Hybridization is an underappreciated evolutionary force.
Hybrid zones often form on the borders between closely related, parapatric populations. % regardless of whether there is a selective advantage to the hybrid phenotype.
Hybridization improves adaptability to changing conditions through new combinations of alleles and increased heterozygosity.
Speciation can also be facilitated by the diversity produced through hybridization, especially if hybrids are dispersed to new regions and new ecological niches.
For example, two species that were born out of hybrid zones are \textit{Pinus densata} and \textit{Senecio squalidus} (Abbott et al. 2014).
%Hybridization can have a substantial influence on whether native species can survive invasion of related species. 
%n some cases genetic mixing can remove most or all pure members of a species like with red wolves \textbf{(cite)} but sometimes a native species survival of an invasion is dependent on introgression from the invader like in the case of californian salamanders \textbf{(cite)}.
The evolutionary history of teosinte (\emph{i.e.}, wild maize) presents a number of compelling opportunities to study hybridization.
Although teosinte represents all wild relatives of maize in the genus {Zea}, for the purposes of this document \textit{Zea mays} ssp. \textit{parviglumis} (hereafter parviglumis) and \textit{Zea mays} ssp. \textit{mexicana} (hereafter mexicana) in aggregate will be called teosinte. 
Parviglumis and mexicana diverged recently, yet there are clear signs of local adaptation within these subspecies as well as ongoing hybridization between them in hybrid zones (Fukunaga et al. 2005).
Habitats of parviglumis and mexicana are easily distinguished based on altitude with hybrids inhabiting intermediate regions.
Hybrid zones defined by altitudinal gradients are particularly interesting because of the dramatic differences in several environmental factors within a small area.
Some of these environmental factors are temperature, atmospheric pressure, soil moisture, day length, light intensity, and wind velocity (Korner 2007).
Teosinte hybrid zones have never been deeply studied before, although one hybrid population was mentioned and identified as admixed (Pyhajarvi et al. 2013)  and others have been briefly mentioned (Fukunaga et al. 2005).
%The genome size and complexity of teosinte is roughly average for angiosperms, making it a good representative of this clade (Gregory 2007). 

I have identified three zones of clustered Mexican hybrid populations between lowland parviglumis and highland mexicana in the Central Plateau of Mexico (CPM) and in the northern and southern Balsas River Valley of Mexico (BRVM). 
To identify hybrids, I analyzed an available Single Nucleotide Polymorphism (SNP) dataset  (Heerwaarden et al. 2010, Heerwaarden et al. 2011, Fang et al. 2012) including 983 SNPs genotyped in 2,793 individuals using the program STRUCTURE.
This program uses a model-based approach to infer population structure and assign individuals to populations using multilocus genotype data (Pritchard et al. 2000).
Hybrids are defined here as samples with majority attribution to one teosinte and a significant ($\geq$ 25\%) attribution to the other.
I plan to confirm these identifications using Reich's f$_{\text{3}}$ ancestry estimation method (Reich et al. 2011).  
This method provides SNP-by-SNP estimates of admixture, and unlike STRUCTURE, is robust to ascertainment bias, which is likely an issue with our dataset as the SNPs were originally developed in a small panel of individuals.

To better understand these hybrids I have formed 3 objectives and associated hypotheses: 
\begin{enumerate} 
	\item Determine how parviglumis-mexicana hybrids are distributed across Mexico.
	\item Ascertain the relationship of these hybrid populations with each other and their neighboring teosinte.
		\begin{description} \item[hypothesis] Conserved, genome-wide patterns of admixture in distinct hybrid zones can be clearly distinguished \end{description}
	\item Explore whether these zones contain stable, locally adapted populations or are simply a product of ongoing hybridization in a tension zone between subspecies.
		\begin{description} \item[hypothesis] The degree of ancestry from each teosinte varies across populations in a manner that is locally adaptive \end{description}
\end{enumerate} 

\hfill\break \textbf{\underline{Objective 1: How extensive are teosinte hybrid zones?} } 
In addition to the SNP dataset that is already available, I would like to generate Genotyping-By-Sequencing data (GBS) of hybrid individuals as well as members of nearby populations of teosinte to answer more in-depth questions about the history of these hybrid zones as well as the nature and degree of admixture amongst the hybrids and between the hybrid populations and their neighbors.  
While we could use the same SNP-chip used to obtain our existing SNP dataset on new samples, GBS provides much higher resolution data at a lower cost.
In order to acquire the GBS data I will first sample more extensively in hybrid zones.  
Due to the high crime rates in both BRVM regions and the recent Guerrero student massacre I will only collect samples in the CPM, leaving sampling in more dangerous areas to our colleagues working in Mexico who have more local knowledge and experience working in these regions.

\hfill\break \textbf{\underline{Objective 2: Is the genomic architecture of hybridization conserved?} } To determine the relationships amongst hybrid populations, I plan to first determine patterns of diversity and differentiation across hybrid populations including both within and between hybrid zone comparisons.
These analyses will provide a rough idea of within and between zone relatedness.
I also plan to use the D statistic (Durand et al. 2011) to determine whether these hybrids are sister to either teosinte subspecies, ancestral to both, or are true hybrids showing equal clustering with each teosinte.
I will also use the GBS data to see whether the genetic architecture of hybridization and introgression varies amongst populations within and between zones.
Understanding the genetic architecture could reveal whether there are conserved regions amongst different zones or at particular altitudes suggesting that they have been selected upon within that niche.

\hfill\break \textbf{\underline{Objective 3: Is the hybrid phenotype adaptive?} }To answer my question about the origin and stability of the hybrid zones, one measure I plan to use is the relative fitness of the hybrids based on a common garden experiment in the CPM. 
If these zones contain stable, locally adapted populations that are fitting into a niche they should not only be present in the intermediate altitude, but also be more fit there and be less fit in higher and lower altitudes relative to mexicana and parviglumis respectively.
I would also like to use HapMix (Price et al. 2009) with our GBS data to build a histogram of the lengths of ancestry segments.
As ancestry segments of a hybrid individual should break up over time due to recombination, if these populations resulted from ancient admixture proceeded by roughly consistent resistance to gene flow there should be a peak of ancestry segments near a specific length.
If these hybrids are the result of ongoing hybridization and gene flow from parent populations the histogram should look roughly like an exponential decay graph as most, but not all, ancestry segments would be highly broken up by recombination.

To build a tree of these hybrids and proximal teosinte populations I plan to use the software Treemix (Pickrell et al. 2012).
Phylogenetic trees, while useful, are often an oversimplification of the relationships between populations. 
Treemix improves upon these trees by adding directed and weighted migration edges. 
Along with a STRUCTURE analysis of these groups and their neighboring teosinte, these analyses should paint a clear picture of the introgression history amongst these hybrid groups including whether their ancestry was originally more parviglumis or mexicana and whether some hybrid populations are derived from others.

\hfill\break \textbf{\underline{Resources Needed:} } To successfully complete this project I will require funding for my salary, for GBS, and for travel to Mexico.  I will also require the assistance of my major professor Dr. Matthew Hufford as well as our collaborators at UC Davis and in Mexico.
A benefit of my work being largely computational and using an available dataset is low data generation costs.

\hfill\break \textbf{\underline{Broader Impacts:} }
To reach out to the greater community I plan to participate in Iowa State University's GK12 program.  Through the program I will be sharing my research with children from the Des Moines public school system.  
I expect that my work will interest these students as Iowa's agriculture is dominated by corn.
In terms of representing ethnic minorities, The Des Moines public school system is the most diverse in the state of Iowa.  This program is therefore a great opportunity to get schoolchildren, including underprivileged groups, excited about STEM research.  
%I believe that I am uniquely capable of answering these questions because I have experience in programming and computational analyses, experienced advisors and collaborators, a solid academic foundation in genomics and the drive and curiosity to stay focused on the project.

%Bibliography
\hfill\break \textbf{\underline{Works cited:}}
\small{ 
\textbf{1: Abbott} and Brennan (2014). \textit{Royal Society Publishing}, doi: 10.1098/rstb.2013.0346
\textbf{2: Durand} et al. (2012). \textit{Molecular Biology and Evolution}, doi: 10.1093/molbev/msr048.
\textbf{3: Fang} et al. (2012). \textit{Genetics}, doi: 10.1534/genetics.112.138578. 
\textbf{4: Fukunaga} et al. (2005). \textit{Genetics}, doi: 10.1534/genetics.104.031393. 
%\textbf{4: Gregory} et al. (2007). \textit{Nucleic Acids Research}, doi:10.1093/nar/gkl828. 
\textbf{5: Heerwaarden} et al. (2010). \textit{Molecular Ecology}, 19(6) 1162-1173. 
\textbf{6: Heerwaarden} et al. (2011). \textit{PNAS}, 108(3) 1088-1092.  
%\textbf{7: Hufford} et al. (2013). \textit{PLOS Genetics}, DOI: 10.1371/journal.pgen.1003477
\textbf{7: Korner} (2007). \textit{Trends in Ecology and Evolution.}, doi:10.1016/j.tree.2007.09.006.
%\textbf{8: Masterson} (1994). \textit{Science}, 264(5157) 421-424.  
\textbf{8: Pickrell} and Pritchard (2012). \textit{PLOS Genetics}, DOI: 10.1371/journal.pgen.1002967. 
\textbf{9: Price} et al. (2009). \textit{PLOS Genetics}, DOI: 10.1371/journal.pgen.1000519. 
\textbf{10: Pritchard} et al. (2000). \textit{the Genetics Society of America}, 155: 945-959.  
\textbf{11: Pyhajarvi} et al. (2013). \textit{Genome Biology and Evolution}, 264(5157) 421-424.  
\textbf{12: Reich} et al. (2011). \textit{The American Journal of Human Genetics}, 89 516-528.

}

\end{document}  